src/Graph.ts

```diff
@@ -34,6 +36,9 @@ import relativeId, { getAliasName } from './utils/relativeId';
 import { timeEnd, timeStart } from './utils/timers';
 import transform from './utils/transform';
 
+// clear plugin cache items after 10 builds unnaccessed
+const PLUGIN_CACHE_EVICTION_EXPIRY = 10;
```

Comment:
It would be good to see this number configurable; CI machines might build dozens of branches per day may benefit from larger numbers so they aren't evicting cache quite as much.

Reply:
@keithamus would that be better as an environment variable, or configuration option?

Reply:
I guess a configuration option - considering pretty much everything else is. With a rollup.config.js it's relatively easy to expose it as a environment variable.

Reply:
I've added a `cacheExpiry` configuration option.